src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,12 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: $transition-speed;
```

Comment:
It is better to specify the property you are applying transition to. Same in header.scss.
```
transition: 
   transform $transition-speed,
   color $transition-speed;
```

Reply:
+1